action.yaml

```diff
@@ -55,6 +55,14 @@ inputs:
       variable GITHUB_SHA.
     required: false
     type: string
+  charm_dir:
+    description: |
+      Relative name of the directory where the metadata.yaml or charmcraft.yaml files are located
+      if they are not in the root directory of the repository. The docs directory is also located
+      under this directory.
+    default: ''
+    required: false
+    type: string
```

Comment:
We should probably also add an e2e test for this here for the new input please: https://github.com/canonical/gatekeeper-repo-test

Reply:
Let's try and get it working, just please include a link here to a successful run on a branch and then we can merge this before the change to the e2e test

Reply:
There are e2e tests that test the new functionality in here:
- https://github.com/canonical/gatekeeper-repo-test/actions/runs/8505195956
- https://github.com/canonical/gatekeeper-repo-test/actions/runs/8505195954